README.md

```diff
@@ -136,6 +136,7 @@ Karamaan Group | United States, NY, New York | Finance | ?
 [Oracle Labs](https://labs.oracle.com/) | Remote | Blockchain research | [Github](https://github.com/erkmos/haskell-companies/pull/29)
 [Permutive](https://permutive.com) | United Kingdom, London | Analytics | [Job Ad](https://permutive.com/jobs/software-engineer-haskell-scala/)
 [Picus Security](https://www.picussecurity.com) | Turkey, Ankara | Information security | [Github](https://github.com/picussecurity?language=haskell)
+[Platonic.Systems](https://platonic.systems) | Remote | Consulting | [GitLab](https://git.platonic.systems) | Yes
```

Comment:
@Fresheyeball https://git.platonic.systems is not publicly accessible when not logged in. At least, accessing it in incognito window redirects me to Google sign-in page.

Reply:
https://gitlab.com/platonic is this the correct public url @Fresheyeball ?

Reply:
@erkmos both are public

Reply:
@erkmos @srid I pushed a commit to have both

Reply:
I removed the platonic instance since it is google account-walled, hope you don't mind